pkg/minikube/driver/endpoint.go

```diff
@@ -30,6 +32,10 @@ func ControlPlaneEndpoint(cc *config.ClusterConfig, cp *config.Node, driverName
 		port, err := oci.ForwardedPort(cc.Driver, cc.Name, cp.Port)
 		hostname := oci.DefaultBindIPV4
 		ip := net.ParseIP(hostname)
+		if ip == nil {
+			errMsg := fmt.Sprintf("failed to parse ip")
+			err = errors.Wrap(err, errMsg)
```

Comment:
* Is Wrap correct here?

* I'd like to add additional context, but from my testing the `ParseIP` fails due to an empty `hostname`. So I'm not sure what I can add that is helpful.

Reply:
instead of error.wraap
you can use fmt.Errorf("failed to parse ip for %s", hostname)